### PR TITLE
Configure MXN currency for company #19

### PR DESCRIPTION
**Pull request by JesusZapata**
Referring to Configure https://github.com/Vauxoo/enterprise-mx/issues/19

### CODE REVIEW

**Comment by JesusZapata** [comment]
Check _Delete all currency rates_
image

Check _Enable multi-currency check from invoice settings_
image

File: l10n_mx_enterprise/demo/global.xml [hunk 1]
```diff
@@ -30,4 +30,11 @@
             ]"/>
     </record>
 
+    <record id="base.group_user" model="res.groups">
+        <field name="implied_ids" eval="[
+            (4, ref('base.group_multi_currency')),
+            (4, ref('product.group_sale_pricelist'))
```
Comment (moylop260) [inline l10n_mx_enterprise/demo/global.xml]: Better run the action from settings enabling multicurrency and apply.

**Comment by JesusZapata** [comment]
I have enabled it since `Invoincing > Configuration > Setting`
Is that what you mean?

image

**Comment by moylop260** [comment]
Yeah, that view is a `*.settings` model.
Then you need enable that field explicit with a: multi_currency=True and `save`

Please, ask to @hugho-ad about

**Comment by JesusZapata** [comment]
Done 

image

File: l10n_mx_enterprise/demo/global.xml (hunk 1 above)
Comment (JesusZapata) [inline l10n_mx_enterprise/demo/global.xml]: Done !

**Comment by moylop260** [comment]
Could you help me to fix new conflicts, please?

**Comment by JesusZapata** [comment]
@moylop260 
Solved the conflict!

**Pull request merged**